samples/java_layout/src/main/java/io/customer/android/sample/java_layout/ui/settings/SettingsActivity.java

```diff
@@ -111,52 +100,20 @@ private void setupViews() {
             }
             onBackPressed();
         });
-        binding.deviceTokenInputLayout.setEndIconOnClickListener(view -> {
-            String deviceToken = ViewUtils.getTextTrimmed(binding.deviceTokenTextInput);
-            ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
-            ClipData clip = ClipData.newPlainText(getString(R.string.device_token), deviceToken);
-            clipboard.setPrimaryClip(clip);
-            Toast.makeText(this, R.string.token_copied, Toast.LENGTH_SHORT).show();
-        });
-        binding.saveButton.setOnClickListener(view -> saveSettings());
-        binding.restoreDefaultsButton.setOnClickListener(view -> {
-            updateIOWithConfig(CustomerIOSDKConfig.getDefaultConfigurations());
-            saveSettings();
-        });
+        binding.settingsSaveButton.setOnClickListener(view -> saveSettings());
+        binding.settingsRestoreDefaultsButton.setOnClickListener(view -> updateIOWithConfig(CustomerIOSDKConfig.getDefaultConfigurations()));
+        ViewUtils.clearErrorWhenTextedEntered(binding.settingsCdpApiKeyLabel, binding.settingsCdpApiKeyLayout);
+        ViewUtils.clearErrorWhenTextedEntered(binding.settingsSiteIdKeyLabel, binding.settingsSiteIdKeyLayout);
     }
 
     private void setupObservers() {
-
-        binding.deviceTokenTextInput.setText(CustomerIO.instance().getRegisteredDeviceToken());
```

Comment:
Moved to internal settings screen

Reply:
The device token location should move to the main screen under the email.

Reply:
@scotttwittrockcio maybe we can add it there for convenience but also keep it in the internal settings?

Reply:
I don't think it's needed in both places. It's a really useful piece of information when testing push messages. It helps me quickly validate the device id matches the one in fly, and the one I'm using to send a test push to.